ci-scripts/docker/Dockerfile.mme.ci.rhel8

```diff
@@ -70,49 +80,45 @@ RUN yum update -y && \
       procps-ng \
       tcpdump \
       openssl \
+      boost \
+      libicu \
+      libidn \
+      libconfig \
+      lksctp-tools \
       net-tools \
       tzdata && \
-    echo "/usr/local/lib" > /etc/ld.so.conf.d/local-lib.conf && \
-    echo "/usr/local/lib64" >> /etc/ld.so.conf.d/local-lib.conf && \
-    yum clean all -y && \
-    rm -rf /var/cache/yum
+    microdnf clean all -y && \
+    rm -rf /var/cache/yum /var/cache/dnf && \
+    rm -f /etc/pki/entitlement/*pem /etc/rhsm/ca/*pem
```

Comment:
you delete certificates here, but they are still present in a previous layer. Is that as expected?

Reply:
@maxhbr 

True and from a Dockerfile point of view there is nothing we can do about it.

But then if I apply the `--squash` option to `sudo podman build`, I got a single layer and the `COPY` and `rm` steps are now a single layer: extract from `sudo podman image inspect magma-mme:xxx`

```
            {
               "created": "2022-10-04T08:04:08.529531721Z",
                "created_by": "/bin/sh -c #(nop) COPY dir:15a72aa3f80ea478140ebfb2db482ad3674a760361040952c7c8b05e757db8ad in /etc/pki/entitlement ",
                "empty_layer": true
            },
            {
                "created": "2022-10-04T08:04:08.685845324Z",
                "created_by": "/bin/sh -c #(nop) COPY dir:d5f328e24ccd61638af79e82a62986dbd6c5b4236b9997dac7cea57168abaeab in /etc/rhsm ",
                "empty_layer": true
            },
            {
                "created": "2022-10-04T08:04:08.840774181Z",
                "created_by": "/bin/sh -c #(nop) COPY dir:e10e3e190e72661f6fc325f7a851602f34c2ba1bcad64dbfb57de7920773e274 in /etc/rhsm/ca ",
                "empty_layer": true
            },
            {
                "created": "2022-10-04T08:04:08.84081369Z",
                "created_by": "/bin/sh -c #(nop) ENV TZ=Europe/Paris",
                "empty_layer": true
            },
            {
                "created": "2022-10-04T08:04:40.014700974Z",
                "created_by": "/bin/sh -c microdnf update -y \u0026\u0026     microdnf -y install       libubsan       libasan       liblsan       sqlite       psmisc       procps-ng       tcpdump       openssl       boost       libicu       libidn       libconfig       lksctp-tools       net-tools       tzdata \u0026\u0026     microdnf clean all -y \u0026\u0026     rm -rf /var/cache/yum /var/cache/dnf \u0026\u0026     rm -f /etc/pki/entitlement/*pem /etc/rhsm/ca/*pem",
                "empty_layer": true
            },
```

The 2 following `image inspect` log files are from the same workspace build, one without `--squash` option, one with it:

[magma-mme-no-squash.log](https://github.com/magma/magma/files/9703971/magma-mme-no-squash.log)
[magma-mme-with-squash.log](https://github.com/magma/magma/files/9703972/magma-mme-with-squash.log)

Reply:
can you somehow enforce the squashing? If someone builds without that, that person might accidentally share certificates, that should not be distributed.

Reply:
Did you try to use build secrets?

Reply:
Why was this working before without the certs?